### ISSUE TITLE
Charged ZnO defects - NEBs

### ISSUE THREAD

**Issue by VondrakMar** [ddmms/ml-peg#306]
### Summary

This benchmark will be composed of different NEB paths in ZnO wurtzite structures containing an oxygen vacancy. The test set will include two different charge states (0 and 2+), with the 2+ charge state exhibiting a lower migration barrier. Additional tests for differently sized cells may also be included.

### Interactive features

NEB barrier plot

### Category

Physicality 

### Data availability

I will provide starting and final positions in xyz files for different charge states, paths and sizes. Also I have the barrier computed on HSE06 level in FHI-aims with intermediate settings. The barrier is calculated for two cell sizes. The potential training data, and publications corresponding to this test will be published in a following month. 

### Computational cost

Smaller NEB calculations can be done in few minutes for smaller cells, large cells can be potentially more expansive, and needs to be run on GPUs.
